SaraHackle/Capstone-Project
# One active card displays at a time, cannot play game with yourself

#### Changes Made
1. Added file `MyActiveCard.js` to `game` directory.
2. Added file `OppActiveCard.js` to `game` directory.
1. Modified file ``NewGameForm.js` to include `availableUsers` state which populates drop down with users other than the logged in user.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout sh-oneActiveCard
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > When a user or opponent selects a card, the buttons to submit a new card disappear, and the card displays in the game component portion of page. When a user does not have a card selected the buttons reappear and a message displays to play a card. 
>When a user starts a new game, they no longer see their name in the list of options.
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
